test/Api.Tests/ControllerTests/UsersControllerTest.cs

```diff
@@ -31,8 +31,7 @@ public UsersControllerTest()
         /// <summary>
         /// Should return code 202 when an admin is trying to invite new users with new Emails
         /// </summary>
-        //[Fact] - Dependency Injection is not working with XUnit and the issue is still open
-        // Refer: https://github.com/xunit/xunit/issues/687
+        //[Fact] - Dependency Injection is not working with XUnit //https://xunit.github.io/docs/shared-context.html#class-fixture
```

Comment:
It seems to be possible, see e.g. http://stackoverflow.com/questions/35928364/creating-a-testserver-and-using-dependency-injection-with-xunit-and-asp-net-core

**But** as we discussed previously, it is not justifiable to put this much effort into this. Your new comment is misleading though, just change it back to the comment you had before